### ISSUE TITLE
List supported browsers

### ISSUE THREAD

**Issue by dustinhayes** [50onRed/pjax-table#3]
It looks like, with the use of `.bind()` and `Array.isArray()`, IE9+ is supported. Is this indented?

**Comment by mparke** [comment]
Yes, this is intended, will look at adding a support list to the README soon.
